tests/quick_tests/nanoflann.cc

```diff
@@ -0,0 +1,57 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2017 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE at
+// the top level of the deal.II distribution.
+//
+//---------------------------------------------------------------------
+
+// Create a list of points, and compute the minimum distance from some other points
+// to this set, using a kdtree library
+
+#include <deal.II/base/logstream.h>
+#include <deal.II/numerics/kdtree.h>
+
+
```

Comment:
@luca-heltai Judging by an error I got when making the Adol-C quick test, I think that you may be missing a `using namespace dealii;` here. Do you want me to add a patch to my open PR?

Reply:
yes, please do. Notice however that all  quicktests  pass on my machine...

Reply:
Ok, I've pushed the patch. Strange that excluding this line from the Adol-C test resulted in an error, e.g.
```
<>/tests/quick_tests/adolc.cc:62:3: error: use of undeclared identifier 'deallog'; did you mean 'dealii::deallog'?
  deallog << "Error (function 1): " << error_func_1 << std::endl;
```
while you didn't see the same here :-/